src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+
+  if (days < 7 && days >= 3) {
```

Comment:
swap your if's blocks order so you can omit first check
```suggestion
  if (days >= 3) {
```